### PR TITLE
Compile tests

### PR DESCRIPTION
**Pull request by robert-zaremba**
Add tests to the compile command

### CODE REVIEW

File: packages/algob/test/fixture-projects/default-config-project/assets/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a sample README file.
```
Comment (Invertisment) [inline packages/algob/test/fixture-projects/default-config-project/assets/README.md:1]: Can we have info here?

File: packages/algob/src/builtin-tasks/compile.ts [hunk 2]
```diff
@@ -25,15 +26,15 @@ export interface TaskArgs {
 }
 
 function _compile ({ force }: TaskArgs, env: AlgobRuntimeEnv): Promise<void> {
```
Comment (Invertisment) [inline packages/algob/src/builtin-tasks/compile.ts:28]: It would be best to rename this function.

**Review by Invertisment — APPROVED** [review]

File: packages/algob/test/fixture-projects/default-config-project/assets/README.md (hunk 1 above)
Comment (robert-zaremba) [reply packages/algob/test/fixture-projects/default-config-project/assets/README.md:1]: what kind of info? I can add whatever you want.

My intention was just to create some other file (which is not `.teal`) so I can test that only teal files are handled and compiled.

File: packages/algob/src/builtin-tasks/compile.ts (hunk 2 above)
Comment (robert-zaremba) [reply packages/algob/src/builtin-tasks/compile.ts:28]: ah, I forgot it. OK. will do as discussed in the previous PR.

File: packages/algob/test/fixture-projects/default-config-project/assets/README.md (hunk 1 above)
Comment (Invertisment) [reply packages/algob/test/fixture-projects/default-config-project/assets/README.md:1]: Oh. Then this could be the contents of this file.

**Review by amityadav0 — APPROVED** [review]

**Review by Invertisment — APPROVED** [review]

**Pull request merged**